core/domain/feedback_services.py

```diff
@@ -292,20 +447,36 @@ def get_all_threads(exploration_id, has_suggestion):
 
 
 def get_all_thread_participants(exploration_id, thread_id):
-    """Returns set of all participants of a thread."""
+    """Fetch all participants of the given thread
+
+    Args:
+        exploration_id: str.
+        thread_id: str.
+
+    Returns:
+        set of all the participants' author_id
+    """
     return set([m.author_id for m in get_messages(exploration_id, thread_id)])
 
 
 def enqueue_feedback_message_batch_email_task(user_id):
-    """Adds a 'send feedback email' (batch) task into the taskqueue."""
+    """Add a 'send feedback email' (batch) task into the task queue.
 
+    Args:
+        user_id: str. The user to be notified
+    """
     taskqueue_services.enqueue_task(
         feconf.TASK_URL_FEEDBACK_MESSAGE_EMAILS, {'user_id': user_id},
         feconf.DEFAULT_FEEDBACK_MESSAGE_EMAIL_COUNTDOWN_SECS)
 
 
 def enqueue_feedback_message_instant_email_task(user_id, reference):
-    """Adds a 'send feedback email' (instant) task into the taskqueue."""
+    """Adds a 'send feedback email' (instant) task into the task queue.
+
+    Args:
+        user_id: str. The user to be notified
+        reference: FeedbackMessageReference.
```

Comment:
Describe this -- "Information about the message to inform the user about", perhaps?